docs/DeveloperGuide.md

```diff
@@ -541,19 +541,59 @@ The following activity diagram summarizes what happens when a user executes a ne
 
 #### Implementation
 
-Step 1. placeholder
+The overview feature's functionality can be split into two portions:
+* Changing the overview type
+* Automatically updating the overview
+
+The former is an active command call, and is facilitated by the `DengueHotspotTrackerParser#parseCommand()`, `OverviewCommandParser#parse()`, `OverviewCommand#execute()`, and `Model#setOverview()` methods.
+
+The latter is passive behaviour, and is largely facilitated by the `Model#getOverview()` and `OverviewDisplay#updateOverviewDisplay()`methods, called after every successful command.
+
+Both of these `Model` methods are implemented in `ModelManager`. These access implementations of the `Overview` and `Analyst` interfaces and the `DataBin` class, where most of the functionality resides.
+
+Unlike other commands, the GUI display of the overview is maintained by the `OverviewDisplay` class, which represents the overview panel (on the right hand side of the window) and holds many `DataBinCard` instances, similar to how `PersonListPanel` carries `PersonCard` instances.
+
+Given below is an example usage scenario for the active functionality and how the overview mechanism behaves at each step.
+
+It would be helpful to note that `DengueHotspotTrackerParser#parseCommand()` is called from `LogicManager#execute()`, which is in turn called from `MainWindow#executeCommand()` when the user inputs a command via the CLI. Many other mechanisms occur in these methods but , for the sake of brevity, we will only focus on the paths and parts of interest.
```

Comment:
```suggestion
It would be helpful to note that `DengueHotspotTrackerParser#parseCommand()` is called from `LogicManager#execute()`, which is in turn called from `MainWindow#executeCommand()` when the user inputs a command via the CLI. Many other mechanisms occur in these methods, but for the sake of brevity, we will only focus on the paths and parts of interest.
```